.readthedocs.yaml

```diff
@@ -0,0 +1,30 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11" 
+    #"3.10", "3.9"
+
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+   configuration: doc/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+# formats:
+#   - pdf
+#   - epub
+
+# Optional but recommended, declare the Python requirements required
+# to build your documentation
+# See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
+python:
+  install:
+    - requirements: doc/requirements.txt
```

Comment:
The requirements for osm-flex cannot be installed with pip alone. So we have two options here:
1. Adding a conda environment file (for the docs or in general) and telling readthedocs to use Conda/Mamba: https://docs.readthedocs.io/en/stable/guides/conda.html
2. Using a requirements.txt and telling Sphinx automodule to ignore import errors of specific modules (those that cannot be installed via pip): https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports

Reply:
how come this hasn't led to any errors in the documentation so far, then?

Reply:
Ahaaa, I think you can build the documentation without installing all dependencies of osm-flex because Sphinx does not *include* your module but only *parses* it. For CLIMADA, we include the module in the `conf.py`, which then of course requires us to load all dependencies because import errors would occur otherwise. Or am I reading this wrong? @chahank, this might be something we should consider for CLIMADA, since building the documentation there is always a hassle due to the dependency management.